src/test/java/bio/terra/common/iam/AuthenticatedUserRequestTest.java

```diff
@@ -44,48 +39,41 @@
           .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
           .enableDefaultTyping(ObjectMapper.DefaultTyping.NON_FINAL);
 
-  private static final String EmailAddress = "test@example.com";
-  private static final UUID RequestId = UUID.randomUUID();
-  private static final String SubjectId = "Subject";
-  private static final String Token = "0123.456-789AbCd";
+  private static final String EMAIL_ADDRESS = "test@example.com";
+  private static final String SUBJECT_ID = "Subject";
+  private static final String TOKEN = "0123.456-789AbCd";
 
   @Test
-  public void fullBuilder() throws Exception {
-    AuthenticatedUserRequest req =
-        AuthenticatedUserRequest.builder()
-            .setEmail(EmailAddress)
-            .setReqId(RequestId)
-            .setSubjectId(SubjectId)
-            .setToken(Token)
-            .build();
-
-    assertEquals(EmailAddress, req.getEmail());
-    assertEquals(SubjectId, req.getSubjectId());
-    assertEquals(Token, req.getToken());
-    assertEquals(Token, req.getRequiredToken());
-    assertNotNull(req.getReqId());
-    validateJsonSerialization(req);
-  }
-
-  @Test
-  public void emptyBuilder() throws Exception {
-    AuthenticatedUserRequest req = AuthenticatedUserRequest.builder().build();
-    assertNull(req.getEmail());
-    assertNull(req.getSubjectId());
-    assertNull(req.getToken());
-    assertNotNull(req.getReqId());
-    assertThrows(ApiException.class, req::getRequiredToken);
+  public void builder() throws Exception {
+    AuthenticatedUserRequest.Builder builder = AuthenticatedUserRequest.builder();
+
+    // Build fails due to no Email
+    assertThrows(IllegalStateException.class, builder::build);
+    builder.setEmail(EMAIL_ADDRESS);
+
+    // Build fails due to no Subject
+    assertThrows(IllegalStateException.class, builder::build);
+    builder.setSubjectId(SUBJECT_ID);
+
+    // Build fails due to no Token
+    assertThrows(IllegalStateException.class, builder::build);
+    builder.setToken(TOKEN);
```

Comment:
this test would pass even if it was allowed to not set the email or the subject given that the token is set last. Consider going `validRequest.toBuilder().setEmail(null).build()` for each field.

Reply:
Note that I'm reusing the local instance `builder` for this very reason, I did verify the coverage using the IntelliJ run-with-coverage functionality.  On a related note, I'd love tho see OpenClover coverage reports as part of our gradle build artifacts to make this less manual/interactive; I'll look into how to make that happen.

Reply:
I think having tests that cover all paths would be better than relying on code coverage tooling, but not a big deal. Easier to ignore a code coverage warning than a failing test.

Reply:
Hmmm... not sure I follow, as this is covering all paths due to multiple calls on the same builder isntance (created on line 48).  Lets discuss offline.